OpenRA.Mods.Common/Traits/Player/ClassicProductionQueue.cs

```diff
@@ -30,7 +30,7 @@ public class ClassicProductionQueueInfo : ProductionQueueInfo, Requires<TechTree
 		public override object Create(ActorInitializer init) { return new ClassicProductionQueue(init, this); }
 	}
 
-	public class ClassicProductionQueue : ProductionQueue, ISync
+	public class ClassicProductionQueue : ProductionQueue
```

Comment:
You need to get rid of https://github.com/OpenRA/OpenRA/pull/10557/files#diff-910e0462c780643261207e2829e4041dR47 as well then.